samples/doc_writer_sample_output.py

```diff
@@ -413,31 +450,31 @@
             "title": "Extraclasses"
         },
         {
-            "@id": "https://hydrus.com/api/collection_2",
+            "@id": "https://hydrus.com/api/vocab#dummyclasses",
             "@type": "Collection",
             "description": "This collection comprises of instances of dummyClass",
             "manages": {
-                "object": "https://hydrus.com/api/dummyClass",
+                "object": "https://hydrus.com/api/vocab#dummyClass",
                 "property": "rdf:type"
             },
             "subClassOf": "http://www.w3.org/ns/hydra/core#Collection",
             "supportedOperation": [
                 {
-                    "@id": "https://hydrus.com/api/collection_2/dummyclasses_retrieve",
+                    "@id": "https://hydrus.com/api/vocab#dummyclasses_retrieve",
```

Comment:
This is again wrong. The @id here needs to be https://hydrus.com/api/collection_2:dummyclasses_retrieve. Here the @id refers to the endpoint where the operation is done, not the definition of the operation.